src/main/java/com/shopify/model/MetafieldValueType.java

```diff
@@ -12,7 +15,11 @@ private MetafieldValueType(final String value) {
 	}
 
 	public static MetafieldValueType toEnum(final String value) {
-		if (STRING.toString().equals(value)) {
+		if (SINGLE_LINE_TEXT_FIELD.toString().equals(value)) {
+			return MetafieldValueType.SINGLE_LINE_TEXT_FIELD;
+		} else if (NUMBER_INTEGER.toString().equals(value)) {
+			return MetafieldValueType.NUMBER_INTEGER;
+		} else if (STRING.toString().equals(value)) {
 			return MetafieldValueType.STRING;
 		} else if (INTEGER.toString().equals(value)) {
 			return MetafieldValueType.INTEGER;
```

Comment:
Any reason you didn't add the rest of types here https://shopify.dev/apps/metafields/definitions/types
while we are in here, I think it would be good to just handle now and it's just a mapping. If something returns here that isn't in this enum we will receive errors.